src/parser/WASMParser.cpp

```diff
@@ -2122,6 +2186,9 @@ class WASMBinaryReader : public wabt::WASMBinaryReaderDelegate {
 
     virtual void OnEndExpr() override
     {
+        // unify eqz in a block followed by a JumpIf
```

Comment:
`Combining an eqz at the end of a block followed by a JumpIf cannot be combined because it is possible to jump to the location after eqz`.